asyncapi.yaml

```diff
@@ -0,0 +1,93 @@
+asyncapi: '2.0.0' # Go parser does not support 2.1.0 at this moment.
+info:
+  title: AsyncAPI Event-Gateway demo API
+  version: 1.0.0-alpha
+  description: This API lets users interact with an instance of the [AsyncAPI Event-Gateway](https://github.com/asyncapi/event-gateway).
+defaultContentType: application/json
+servers:
+  asyncapi-event-gateway-demo:
+    url: 'event-gateway-demo.asyncapi.org:28002'
+    protocol: kafka
+    description: Asyncapi [Event-Gateway](https://github.com/asyncapi/event-gateway) demo Kafka proxy. Expects messages based on StreetLights tutorial available at https://www.asyncapi.com/docs/tutorials/streetlights.
+  asyncapi-event-gateway-demo-validation:
+    url: 'event-gateway-demo.asyncapi.org:5000/ws'
+    protocol: ws
+    description: Asyncapi [Event-Gateway](https://github.com/asyncapi/event-gateway) demo. Subscribe for Kafka proxy message validation errors.
+  asyncapi-kafka-test:
+    url: 'asyncapi-kafka-test-asyncapi-8f90.aivencloud.com:20472' # Kafka with 3 brokers.
+    protocol: kafka-secure
+    description: AsyncAPI Kafka test broker. Private.
+    x-eventgateway-dial-mapping: '0.0.0.0:20473,event-gateway-demo.asyncapi.org:20473|0.0.0.0:20474,event-gateway-demo.asyncapi.org:20474|0.0.0.0:20475,event-gateway-demo.asyncapi.org:20475' # Dynamic ports starts at 20473
+channels:
+  event-gateway-demo:
+    description: Demo Kafka topic for asyncapi-event-gateway-demo server. Users can send their events to this topic and see how message validation happens on the fly based on this right AsyncAPI file by connecting to `event-gateway-demo-validation-events` channel (`asyncapi-event-gateway-demo-validation` ws server).
+    subscribe:
```

Comment:
so can users send their events, then why is says subscribe? from a user perspective it means people can subscribe as gateway publishes there

Reply:
You won't believe how much I need this to happen https://github.com/asyncapi/spec/issues/618#issuecomment-918840251 😢